tests/foreman/cli/test_realm.py

```diff
@@ -21,70 +21,62 @@
 import pytest
 from fauxfactory import gen_string
 
-from robottelo.cleanup import capsule_cleanup
 from robottelo.cleanup import realm_cleanup
 from robottelo.cli.base import CLIReturnCodeError
 from robottelo.cli.factory import CLIFactoryError
-from robottelo.cli.factory import make_proxy
 from robottelo.cli.factory import make_realm
 from robottelo.cli.realm import Realm
 
 
-@pytest.fixture(scope='module')
-def _make_proxy(request):
-    """Create a Proxy and register the cleanup function"""
-    proxy = make_proxy()
-    request.addfinalizer(lambda: capsule_cleanup(proxy['id']))
-    return proxy
-
-
 @pytest.mark.run_in_one_thread
 class TestRealm:
     """Tests for Realms via Hammer CLI, must be run on QE Satellite Server.
     Requires enroll_idm() and configure_realm() to configure the test environment."""
 
     @pytest.mark.tier1
-    def test_positive_delete_by_name(self, _make_proxy):
+    def test_positive_delete_by_name(self, module_fake_proxy):
         """Realm deletion by realm name
 
         :id: ef3967e6-d53d-4dec-b74f-c20448d5fc6d
 
         :expectedresults: Realm is deleted
         """
-        realm = make_realm({'realm-proxy-id': _make_proxy['id'], 'realm-type': 'Active Directory'})
+        realm = make_realm(
+            {'realm-proxy-id': module_fake_proxy.id, 'realm-type': 'Active Directory'}
+        )
         Realm.delete({'name': realm['name']})
         with pytest.raises(CLIReturnCodeError):
             Realm.info({'id': realm['id']})
 
     @pytest.mark.tier1
-    def test_positive_delete_by_id(self, _make_proxy):
+    def test_positive_delete_by_id(self, module_fake_proxy):
         """Realm deletion by realm ID
 
         :id: 7c1aca0e-9724-40de-b38f-9189bdae0514
 
         :expectedresults: Realm is deleted
         """
-        proxy = _make_proxy
-        realm = make_realm({'realm-proxy-id': proxy['id'], 'realm-type': 'Active Directory'})
+        realm = make_realm(
+            {'realm-proxy-id': module_fake_proxy['id'], 'realm-type': 'Active Directory'}
```

Comment:
Why item access here when you used attribute access above?

Reply:
ah! didn't complete my copy-paste before marking this RFR, thanks!

Reply:
Updated